app/controllers/qualifications/one_login_users/name_changes_controller.rb

```diff
@@ -34,6 +34,18 @@ def show
         @name_change = current_user.name_changes.find(params[:id])
       end
 
+      def confirm
+        @name_change = current_user.name_changes.find(params[:id])
+        reference_number = qualifications_api_client.send_name_change(name_change: @name_change)
```

Comment:
Should reference number be named case number; as thats what it is in the API? Should we consider changing the name of the send name change method if its returning a case number?

Reply:
The TRS API calls it a case number and the AYTQ frontend calls it a reference number. There's a difference in nomenclature. I've opted to use reference number within AYTQ, treating the API client as the boundary between  AYTQ and TRS.